tensorflow/python/keras/saving/save_test.py

```diff
@@ -71,6 +71,13 @@ def test_save_format_defaults(self):
     save.save_model(self.model, path)
     self.assert_saved_model(path)
 
+  @test_util.run_v2_only
+  def test_save_format_defaults_pathlib(self):
+    if sys.version_info >= (3, 6):
```

Comment:
Seems that pathlib is newly added in 3.4, why do we choose 3.6 as the version to check?

https://docs.python.org/3/library/pathlib.html

Reply:
@qlzh727 
Yes I see pathlib is added in 3.4, but it is only loaded here for 3.6 ( https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/saving/save_test.py#L44). Should I also change those?

Reply:
I see. The original PR has more details about why 3.6 is used. Let's stay with 3.6 then.